### DISCUSSION TITLE
How to disable community containers?

### DISCUSSION THREAD

**Discussion by xneakersvault** [nextcloud/all-in-one#4962] [Questions] [answered]
### Discussed in https://github.com/nextcloud/all-in-one/discussions/3679

<div type='discussions-op-text'>

<sup>Originally posted by **rd-911** November  5, 2023</sup>
Hi together, 

I just experimented with some community containers (especially local-ai and pi-hole). As Pi-Hole did not start ("s6-overlay-suexec: fatal: can only run as pid 1", I did not dig deeper into this) and local-ai did not come up with the results I expected, I wanted to revert everything back to my basic install. 

Unfortunately the way back is not as streamlined and easy as adding the community containers. Although I removed the local-ai contianer, then removed the mastercontainer, issued the run command without adding the community container, my AIO configuration page is still showing Local AI in the containers sections and is trying to start it. This is even after a reboot. Now istopped the local ai container manually.

Can someone please hint me in the right direction which configuration file to touch, so the community container is gone for good in my setup?

Thanks and kind regards
RD</div>

I am currently having the same issue, i have AIO running on docekr compose, i removed the variable for the community container, as stated here https://github.com/nextcloud/all-in-one/tree/main/community-containers#how-to-remove-containers-from-aios-stack, when i run my docker compose back up, the removed container appears on the AIO webinterface and when I start the containers, it pulls the image back and starts the container that i removed. This is with Face Recognition, any idea on how to fully remove it?

**Comment by szaimen** [comment, marked as answer]
Hi, as written in the docs simply removing the variable is not the correct way. Did you try `--env AIO_COMMUNITY_CONTAINERS=" "` ?

**Comment by xneakersvault** [comment]
ok, since I am using docker compose, i had to set AIO_COMMUNITY_CONTAINERS= without the "" in order to get it to work. Hope this helps someone in the future

**Reply by MichaKersloot** [reply]
Shure helped me..
